src/thunderbots/software/ai/primitive/catch_primitive.h

```diff
@@ -8,6 +8,7 @@ class CatchPrimitive : public Primitive
 {
    public:
     static const std::string PRIMITIVE_NAME;
+    static constexpr double STANDARD_INTERCEPT_MARGIN = 1.0;
```

Comment:
This should go in the visitor implementation for this class, since it is specific to the GRSim primitive implementation.

Reply:
Ok. I will move it.